app/package.json

```diff
@@ -110,10 +108,28 @@
     "webpack": "^5.35.0",
     "webpack-bundle-analyzer": "4.4.2",
     "webpack-chain": "6.5.1",
-    "webpack-dev-server": "4.0.0",
+    "webpack-dev-server": "4.1.1",
     "webpack-merge": "5.8.0",
     "webpack-node-externals": "3.0.0"
   },
+  "devDependencies": {
+    "electron-builder": "22.4.0",
+    "electron-packager": "^15.3.0",
+    "quasar": "^2.0.4",
+    "vuex": "^4.0.0"
+  },
+  "peerDependencies": {
+    "electron-builder": "^22.0.0",
+    "electron-packager": "^15.0.0"
```

Comment:
I think we are definitely supporting these versions, but we need to take a look to see if we are already supporting or want to support lower versions.